packages/server/src/services/database/types.ts

```diff
@@ -337,6 +318,26 @@ export interface TaskState extends WithDates {
 	enabled?: number;
 }
 
+export interface User extends WithDates, WithUuid {
+	email?: string;
+	password?: string;
+	full_name?: string;
+	is_admin?: number;
+	email_confirmed?: number;
+	must_set_password?: number;
+	account_type?: number;
+	can_upload?: number;
+	max_item_size?: number | null;
+	can_share_folder?: number | null;
+	can_share_note?: number | null;
+	max_total_item_size?: number | null;
+	total_item_size?: number;
+	enabled?: number;
+	disabled_time?: number;
+	can_receive_folder?: number;
+	is_external?: number;
+}
+
```

Comment:
Not sure why the code was moved but could you please put it back where it was? As it makes it hard to view the diff, and it makes future merges also more complicated.